### ISSUE TITLE
Mesmerizer networking breaks

### ISSUE THREAD

**Issue by rbergen** [PlummersSoftwareLLC/NightDriverStrip#258]
## Bug report

**Problem**
The Mesmerizer board stops responding to incoming requests (like to the webserver), and is also unable to make any outbound connections henceforth (as shown by the logging). Logging shows WiFi connectivity to still be active, although the board does not respond to any incoming network requests any more. A reboot is required to restore functional networking.

**Steps**
1. Start the board 
2.  Issue a number of webserver requests in short succession
3. At some point, the browser (Chrome) starts showing the message "This site can't be reached, &lt;IP address&gt; took too long to respond."

**Notes**
This is a log snippet that captured an exact moment the networking stopped working:

```text
(W) (Draw)(C1) Spawning thread to check weather..
(I) (getWeatherData)(C0) Got today's temps: Now 14 Lo 12, Hi 15, Icon 1
(W) (UpdateWeather)(C0) Got today's weather
WiFi: WL_CONNECTED, IP: 192.168.1.155, Mem: 34836, LargestBlk: 30708, PSRAM Free: 1028947/4175963, LED FPS: 23 Refresh: 63 Hz, Audio FPS: 62, MinVU:  350.4, PeakVU:  361.8, VURatio: 0.0 Buffer: 0/500, CPU: 051%, 069%, FreeDraw: 0.016
(W) (getTomorrowTemps)(C0) Error fetching forecast data for location: Vianen in country: nl
(W) (UpdateWeather)(C0) Weather thread exiting
```

The first outgoing HTTP request succeeded, the second failed. From this point onwards, inbound and outgoing networking was broken.
 
I have tried to also reproduce the issue on an M5StickC Plus with the Spectrum project, but was unable to do so.

**Comment by rbergen** [comment]
While going through the logging on my running monitoring session, I came across the following lines:

```text
[7502013][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
WiFi: WL_CONNECTED, IP: 192.168.1.226, Mem: 46980, LargestBlk: 30708, PSRAM Free: 1026211/4175931, LED FPS: 27 Refresh: 50 Hz, Audio FPS: 62, MinVU:  303.9, PeakVU:  333.3, VURatio: 0.0 Buffer: 0/500, CPU: 054%, 094%, FreeDraw: 0.002
```

Apparently error 12 is ENOMEM, which could well mean the issue is caused by something running out of memory. With about 46K remaining (and a largest block of 30K), I can imagine this indeed being the case.

**Comment by rbergen** [comment]
I'm closing this, as I've been unable to observe this behavior over the past number of days. My best guess is that this issue got fixed with the upgrade from espressif 6.10 to version 6.20.
